src/rollback_to_stable/rts_btree.c

```diff
@@ -83,8 +83,8 @@ __rts_btree_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *firs
               WT_RTS_VERB_TAG_STABLE_UPDATE_FOUND
               "stable update found with txnid=%" PRIu64
               ", stable_timestamp=%s,  durable_timestamp=%s, flags 0x%" PRIx16,
-              upd->txnid, __wt_timestamp_to_string(rollback_timestamp, ts_string[1]),
```

Comment:
Is line 69 (above this code) correct?

Reply:
Actually there is no issue in using `ts_string[1]` as only one string is used in this message, but I changed it to use the 0th array instead of the 1st array. Yes, it can be changed for code at line 69.